WebRoot/css/login.css

```diff
@@ -46,47 +46,48 @@ FORM {
 .signIn {
 	font-size: 20px;
 	padding: 19px 40px;
-	color: #4d4d4d;
+	color: @gray-darker@;
```

Comment:
Can we correct spell at all places for gray to grey

Reply:
done